projects/ngx-openlayers/src/lib/sources/imagestatic.component.ts

```diff
@@ -30,12 +30,25 @@ export class SourceImageStaticComponent extends SourceComponent implements OnIni
   @Input()
   imageSize?: Size;
 
+  @Output()
+  onImageLoadStart: EventEmitter<source.ImageEvent>;
+  @Output()
+  onImageLoadEnd: EventEmitter<source.ImageEvent>;
+  @Output()
+  onImageLoadError: EventEmitter<source.ImageEvent>;
+
   constructor(@Host() layer: LayerImageComponent) {
     super(layer);
+    this.onImageLoadStart = new EventEmitter<source.ImageEvent>();
+    this.onImageLoadEnd = new EventEmitter<source.ImageEvent>();
+    this.onImageLoadError = new EventEmitter<source.ImageEvent>();
```

Comment:
@Yakoust I still think we should not initialize this here but directly in the field. Can we start to do this for new elements and move old ones one by one (my favorite boy scout strategy).

Reply:
Yes agree with that :+1:

Reply:
I've just fixed it and pushed.